packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -227,6 +224,19 @@ std::vector<std::vector<Scalar> > generate_fem1d_element_values() {
   return mat;
 }
 
+template<class Scalar, class Node>
+Kokkos::View<Scalar*[2], Kokkos::LayoutLeft, typename Node::device_type > generate_fem1d_element_values_kokkos() {
+  Kokkos::View<Scalar*[2], Kokkos::LayoutLeft, typename Node::device_type > mat;
+
+  Kokkos::resize(mat, 2);
```

Comment:
If you use the type `View<Scalar[2][2], ...>`, then you won't have to resize (in fact, you won't be able to, which is good because it's a 2 x 2 thing).